manifests/vc/standalone_test.pp

```diff
@@ -123,6 +141,14 @@
     description      => 'VC-standalone service',
   }
 
+  file { '/opt/vc/tls-readme.txt':
+    ensure  => file,
+    mode    => '0744',
+    owner   => 'root',
+    group   => 'root',
+    content => template('sunet/vc/standalone/test/tls-readme.txt.erb'),
```

Comment:
tls-readme.txt.erb does not exists, add .erb.

What's the purpose of the readme?